src/mlpack/methods/ann/gan/gan.hpp

```diff
@@ -101,11 +101,13 @@ class GAN
 
   /**
    * Train function.
-   *
+   * 
+   * @tparam CallbackTypes Types of Callback functions.
```

Comment:
I know you haven"t add the parameter but can you add a description for the `OptimizerType` as well? You can copy the description from:

 https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/logistic_regression/logistic_regression.hpp#L125

Reply:
done.